docs/docs/tutorials/orca_k8s/orca_k8s_tutorial.md

```diff
@@ -0,0 +1,563 @@
+```python
+from bigdl.orca import stop_orca_context
+
+stop_orca_context()
+```
+
+For more details, please see [OrcaContext](https://bigdl.readthedocs.io/en/latest/doc/Orca/Overview/orca-context.html).
+
+## 1.2 K8s Client&Cluster
+The difference between k8s-client and k8s-cluster is where you run your Spark driver. 
+
+For k8s-client, the Spark driver runs in the client process (outside the K8s cluster), while for k8s-cluster the Spark driver runs inside the K8s cluster.
+
+Please see more details in [K8s-Cluster](https://spark.apache.org/docs/latest/running-on-kubernetes.html#cluster-mode) and [K8s-Client](https://spark.apache.org/docs/latest/running-on-kubernetes.html#client-mode).
+
+
+## 1.3 Load Data from Network File Systems (NFS)
+When you are running programs on K8s, please load data from a network file system (NFS) instead of the local file system.
+
+After mounting the NFS into BigDL container (see __Section 2.2 & 4__), the Fashion-MNIST example could load data from NFS the same as loading from a local file system.
+
+```python
+import torch
+import torchvision
+import torchvision.transforms as transforms
+from bigdl.orca.data.file import get_remote_file_to_local
+
+def train_data_creator(config, batch_size):
+    transform = transforms.Compose([transforms.ToTensor(),
+                                    transforms.Normalize((0.5,), (0.5,))])
+
+    trainset = torchvision.datasets.FashionMNIST(root="/bigdl/nfsdata/dataset", train=True, 
+                                                 download=False, transform=transform)
+
+    trainloader = torch.utils.data.DataLoader(trainset, batch_size=batch_size, shuffle=True, num_workers=0)
+    return trainloader
+```
+
+# 2. Create & Lunch BigDL K8s Container 
+## 2.1 Pull Docker Image
+Please pull the prebuilt BigDL `bigdl-k8s` image from [Docker Hub](https://hub.docker.com/r/intelanalytics/bigdl-k8s/tags) as follows:
+```bash
+sudo docker pull intelanalytics/bigdl-k8s:latest
+```
+
+__Note:__
+* The latest BigDL image is built on top of Spark 3.1.2.
+
+## 2.2 Create a K8s Client Container
+Please launch the __Client Container__ following the script below:
+```bash
+sudo docker run -itd --net=host \
+    -v /etc/kubernetes:/etc/kubernetes \
+    -v /root/.kube:/root/.kube \
+    intelanalytics/bigdl-k8s:latest bash
+```
+
+In the script:
+* `--net=host`: use the host network stack for the Docker container;
+* `-v /etc/kubernetes:/etc/kubernetes`: specify the path of kubernetes configurations;
+* `-v /root/.kube:/root/.kube`: specify the path of kubernetes installation;
+
+__Note:__
+* The __Client Container__ contains all the required environment and libraries except Hadoop/K8s configs.
+* You needn't to create an __Executor Container__ manually, which is scheduled by K8s at runtime.
+
+
+We __highly recommand__ you to specify more arguments when using `spark-submit` script:
```

Comment:
recommand -> recommend
spark-submit script -> spark-submit command to run your application